frontend/src/libs/components/button/styles.module.scss

```diff
@@ -0,0 +1,13 @@
+.btn {
```

Comment:
```suggestion
.button {
```
Abbreviations are prohibited according to the rule B16

Reply:
done